libs/live-wallet/src/store.ts

```diff
@@ -7,18 +7,31 @@ import { Account, AccountLike, AccountRaw, AccountUserData } from "@ledgerhq/typ
 import { getDefaultAccountName, getDefaultAccountNameForCurrencyIndex } from "./accountName";
 import { AddAccountsAction } from "./addAccounts";
 import { getCryptoCurrencyById } from "@ledgerhq/cryptoassets/currencies";
+import { AccountDescriptor } from "./walletsync/datatypes/accounts";
+import type { WalletSyncAccountsUpdate, WalletSyncDiff, WalletSyncState } from "./walletsync/state";
 
 export type WalletState = {
   // user's customized name for each account id
   accountNames: Map<string, string>;
 
   // a set of all the account ids that are starred (NB: token accounts can also be starred)
   starredAccountIds: Set<string>;
+
+  /////
+  // FIXME this part of the state is not correctly saved/restored at the moment, we need to improve the INIT_ACCOUNTS to include these data!
+
+  // local copy of the wallet sync data last synchronized with the backend of wallet sync, in order to be able to diff what we need to do when we apply an incremental update
+  wsState: WalletSyncState;
+
+  // list of account ids that are not imported in the wallet sync data, because we need to be able to retry them later. possibly on startup.
+  wsStateNonImportedAccountIds: string[];
```

Comment:
TODO: we need a way on the web tools to fake a non supported coin and be able to test this.